### ISSUE TITLE
Compiling fails on M1 with sbcl

### ISSUE THREAD

**Issue by paines** [lispgames/cl-sdl2#147]
So, I was trying to pull sketch via Quicklisp, which relies on SDL2 on an M1 Mac. There was an issue regarding c2ffi which I managed to fix by compiling it manually described in https://github.com/rpav/c2ffi/issues/104

Now when trying to compile sketch it fails at 
```
To load "sketch":
  Load 1 ASDF system:
    sketch
; Loading "sketch"
; 
; caught ERROR:
;   READ error during COMPILE-FILE:
;   
;     Symbol "+SDL-PIXELFORMAT-UNKNOWN+" not found in the SDL2-FFI package.
;   
;       Line: 4, Column: 69, File-Position: 106
;   
;       Stream: #<SB-INT:FORM-TRACKING-STREAM for "file /Users/aneslihovac/quicklisp/dists/quicklisp/software/cl-sdl2-20211230-git/src/constants.lisp" {7009C700A3}>
```

I am unsure if the issue really is related to cl-sdl2, autowrap or c2ffi . If I am totally wrong here, please excuse my ignorance.

**Comment by iamFIREcracker** [comment]
@paines - take a look at: https://github.com/lispgames/cl-sdl2/issues/154#issuecomment-1280030566

I am going to try and get relevant changes merged upstream, so it would be possible to run Sketch on newer mac without fiddling with forks and whatnot.

**Comment by fitzsim** [comment]
This should be fixed by #167.  Please reopen if it is not.
